src/test/kotlin/net/maizegenetics/phgv2/cli/StartServerTest.kt

```diff
@@ -0,0 +1,118 @@
+import org.junit.jupiter.api.AfterAll
+import org.junit.jupiter.api.BeforeAll
+import java.io.File
+import java.nio.file.Files
+import java.nio.file.Paths
+import kotlin.test.Test
+import kotlin.test.assertTrue
+
+/**
+ * It is difficult to test the StartServer command. When we start the server, it doesn't return
+ * until the server is stopped.
+ * Testing of the dbPath variable depends on checking values in the application.conf file.
+ * We identify the location of the application.conf file based on the location of the StartServer class.
+ * For non-junits, this will be in <>/phg/resources/application.conf.  But for junit it is in the
+ * repository at <>phg/cli/resources/application.conf, so this code fails.
+ *
+ * That leaves us with the option of testing a couple of the functions in the StartServer class,
+ * so that is what is included here.
+ *
+ */
+class StartServerTest {
+    companion object {
+
+        @JvmStatic
+        @BeforeAll
+        fun copyApplicationConfFile() {
+            // copy the application.conf file to a saved location
+            // in the same folder.
+            val appHome = StartServer().getClassPath().substringBefore("classes")
+            val configPath = Paths.get("${appHome}/resources/main/application.conf")
+
+            var config = Files.readString(Paths.get(configPath.toString()))
+
+            val configSaved = Paths.get("${appHome}/resources/main/application.conf.saved")
+            //write the new config file - do I have permission ???
+            configSaved.toFile().writeText(config.toString())
+
+            // Now, for the real file, we need to re-write it with the TILEDB_URI line removed.
+            // When CI tests are run, a script writes the TILEDB_URI line to the application.conf file.
+            // Copy the original config file with the TILEDB_URI line removed back to the original location.
+            config = ""
+            Files.lines(Paths.get(configPath.toString())).forEach { line ->
+                if (!line.startsWith("TILEDB_URI")) {
+                    config += line + "\n"
+                }
+            }
+            //write the new config file
+            configPath.toFile().writeText(config.toString())
+        }
+
+        @JvmStatic
+        @AfterAll
+        fun returnApplicationConfFile() {
+            val appHome = StartServer().getClassPath().substringBefore("classes")
+            val savedConfigPath = Paths.get("${appHome}/resources/main/application.conf.saved")
+
+            var config = Files.readString(Paths.get(savedConfigPath.toString()))
+
+            val configOrig = Paths.get("${appHome}/resources/main/application.conf")
+            //return the original config file
+            configOrig.toFile().writeText(config.toString())
+
+            // delete the saved config file
+            savedConfigPath.toFile().delete()
+
+        }
+    }
+
+    // This test should not be run as it results in the software hanging while the
```

Comment:
Why don't you just delete the test if it is not supposed to be run.

Reply:
I left it in as I sometimes ran it to test whether initial verification data on startup.  But since the distributed tar file has different paths for the application.conf file than does the phg_v2 repository where junits are run, it really isn't that useful.  I'll remove it